mk_verificator/tests/ceph/config.yaml

```diff
@@ -1,2 +1,3 @@
 ---
-ceph_monitors: ['ceph-005*', 'ceph-006*', 'ceph-007*']
+ceph_monitors: ['ceph-003*', 'ceph-004*', 'ceph-005*']
```

Comment:
I suppose we can even move it to the global config, as we use information about ceph monitors not only here